L_3/webpack.config.js

```diff
@@ -0,0 +1,20 @@
+
+const path =  require("path");
+
+module.exports = {
+    entry: "./src/App.js",
+    output: {
+        path: path.resolve(__filename, "dist "),
```

Comment:
```suggestion
        path: path.resolve(__dirname, "dist "),
```
вот отсюда вся беда)